src/react-native/04-intro-to-jsx/intro-to-jsx.md

```diff
@@ -12,25 +12,37 @@ In this section, you will:
 
 - TODO
 
-## Objective 1: TODO
+## Objective 1: Create a UI with JSX
 
-TODO
+Now that we have our project set up, let’s update our page to look like the design below:
+
+<img alt="Screenshot of a mobile application interface text “Place My Order: Coming Soon To...” and “Illinois“." src="../../static/img/react-native/04-intro-to-jsx/01-solution.png" style="max-height: 750px;"/>
 
 ### Concept TODO
 
 TODO
 
 ### Setup 1
 
-TODO
+✏️ Update **src/App.tsx** to be:
+
+@diff ../../../exercises/react-native/03-creating-a-new-app/04-solution/App.tsx ../../../exercises/react-native/04-intro-to-jsx/01-problem/App.tsx only
```

Comment:
There’s no highlighting in the diff, is that right?

Reply:
That is right, there isnt a setup for these objectives

Reply:
Ok, if there’s no setup, we don’t need to include the section.